Module not found: event-emitter

Node version: 8.7.0

Getting this error trying to npm run server:

`ERROR in ./src/client/globalBus.js
Module not found: Error: Can't resolve 'event-emitter' in '/Users/mark/Desktop/background-geolocation-console/src/client'
 @ ./src/client/globalBus.js 1:0-36
 @ ./src/client/components/ListView.js
 @ ./src/client/components/Viewport.js
 @ ./src/client/main.js
 @ multi webpack-hot-middleware/client?path=/__webpack_hmr&timeout=20000 react-hot-loader/patch ./main.js`

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
     "babel-preset-node6": "^11.0.0",
     "classnames": "^2.2.5",
     "compression": "^1.6.2",
+    "event-emitter": "^0.3.5",
     "google-map-react": "^0.23.0",
     "lodash": "^4.17.4",
     "moment": "^2.18.1",

